src/include/os_fhandle.i

```diff
@@ -17,8 +38,7 @@ static inline int
 __wt_directory_sync_fh(WT_SESSION_IMPL *session, WT_FH *fh)
```

Comment:
> The comment above this function doesn't make sense and I'm not sure why we need a separate wrapper for fsync and directory_sync_fh any more. You had a different implementation that called the file_system method directly, but that's not robust with the pluggable file system implementation.

I agree, this function has out-lived its usefulness, I've removed it in b8a5a94.